RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release Process
+
+## Overview
+
+The Karpenter project is an SIG autoscaling project that has the following two components:
+- Kubernetes Custom Resource Definitions (CRDs)
+- Corresponding Go API in the form of [`sigs.k8s.io/karpenter`](https://pkg.go.dev/sigs.k8s.io/karpenter) Go package
+
+This repository is the home for both of the above components.
+
+## Versioning Strategy
+
+The versioning strategy for this project is covered in detail in [the release type section of the compatability documentation](https://karpenter.sh/docs/upgrading/compatibility/#release-types).
+
+## Releasing a New Version
+
+The following steps must be done by one of the [Karpenter Maintainers](https://github.com/kubernetes/org/blob/main/config/kubernetes-sigs/sig-autoscaling/teams.yaml):
+
+For a **MAJOR**, **MINOR**, or **RC** release:
+- Verify the CI tests pass before continuing.
+- Create a tag using the `HEAD` of the `release-vx.x` branch by using `git tag vx.x.x`
```

Comment:
```suggestion
- Create a tag using the `HEAD` of the `release-vx.x.x` branch by using `git tag vx.x.x`
```
There's a few places where this change should be made, not sure if there's a better way to indicate the last x there is literal (e.g. release-v0.33.x) but this should be fine.

Reply:
Nice catch.

Reply:
Nice catch. Updated with `<major>.<minor>.<patch>` syntax